guides/runbooks/orca-zombie-executions.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: single
+title:  "Orca: Zombie Executions"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Aliases: `orphaned execution`
+
+A pipeline or task is "stuck" in a `RUNNING` state but not doing anything.
+
+# Diagnosis
+
+Logs will be emitted regularly for Executions that are currently running in Orca.
+If no logs have been emitted for over 10 minutes for a `RUNNING` Execution, it is very likely a zombie.
```

Comment:
Also, looking at the queue itself can be instructive.

Reply:
If that's the case, I'd like to add an admin API to inspect the queue. I think guiding people to inspect the internals of the queue with Redis-fu is dangerous. I'm happy to add more admin functionality--I think we need it--but I'd like some guidance on what explicitly you're checking. Inspection is also highly implementation-specific. If we add a Dynomite queue backend, SQS or Google Pub/Sub, I have strong doubts inspection would be transferrable or even possible.